MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift

```diff
@@ -37,6 +49,11 @@
  */
 // write your code here
 
+var dearName = "Xixi"
+
+dearName = "Lulu"
+
+morningDear(dearName)
 
 
```

Comment:
5. Write your own function in which you declare a variable (of any type) inside the function's body. Print out this variable to the console from within your function. After you print the variable once, assign it to a new variable on the next line. Print it again (after the line on which you assign it to a new value). Call your function several times. What do you expect to see printed to the playground's console each time you call this function?

// write your code here
func morningVariable() {
    var greeting = "Good morning"
    print("\(greeting), world!")
    greeting = "Good evening"
    print("\(greeting), world!")
}